src/Tipsy.js

```diff
@@ -11,6 +10,27 @@ import PropTypes from 'prop-types';
  * Like bootstrap's tooltip plugin, Tipsy does not rely on images.
  */
 
+class Container extends PureComponent {
+  render() {
+    return (
+      <div className={`Tipsy in ${this.props.placement}`} role="tooltip" >
+        <div className="Tipsy-arrow" />
+        <div className="Tipsy-inner" >{this.props.children}</div>
+      </div>
```

Comment:
What does this container do?

I see some props for placement, but the Tipsy component already seems to accept a `position` prop.

Reply:
Exactly.

Reply:
@JustinRyanH, did this already support that with the `content` prop?

It looks like `content` can be a `node`. Haven't thought about how styling would work with that, but it looks like it would work.

Reply:
Yeah that changes the inside content of the tipsy but not elements of the tipsy itself. Especially if there was a desire to have a more complicated tipsy or customized business needs for tipsy design.

Reply:
Are there cases where changing the content of the tipsy wouldn't be enough? Unless I'm reading it wrong, it seems like we could use arbitrarily complicated content there.